test/byContract/IexecPoco/01_initialize.test.ts

```diff
@@ -0,0 +1,214 @@
+// SPDX-FileCopyrightText: 2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { loadFixture, time } from '@nomicfoundation/hardhat-network-helpers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { ethers, expect } from 'hardhat';
+import { loadHardhatFixtureDeployment } from '../../../scripts/hardhat-fixture-deployer';
+import { IexecInterfaceNative, IexecInterfaceNative__factory } from '../../../typechain';
+import {
+    OrdersAssets,
+    OrdersPrices,
+    buildOrders,
+    createEmptyWorkerpoolOrder,
+} from '../../../utils/createOrders';
+import { TaskStatusEnum, getIexecAccounts, getTaskId } from '../../../utils/poco-tools';
+import { IexecWrapper } from '../../utils/IexecWrapper';
+
+const categoryTime = 300;
+const maxDealDuration = 10 * categoryTime;
+const appPrice = 1000;
+const datasetPrice = 1_000_000;
+const workerpoolPrice = 1_000_000_000;
+
+describe('Poco', async () => {
+    let proxyAddress: string;
+    let iexecPoco: IexecInterfaceNative;
+    let iexecPocoAsAnyone: IexecInterfaceNative;
+    let iexecWrapper: IexecWrapper;
+    let appAddress = '';
+    let workerpoolAddress = '';
+    let datasetAddress = '';
```

Comment:
let appAddress : string;
    let workerpoolAddress : string;
    let datasetAddress : string;
```suggestion
    let appAddress : string;
    let workerpoolAddress : string;
    let datasetAddress : string;
```

Reply:
Can do the same in claim.test

Reply:
Thanks, I propose to declare them in a single line since we assign them in a single line, lmk :ok_hand: [Declare asset addresses with type](https://github.com/iExecBlockchainComputing/PoCo/pull/74/commits/60cfe48f16c9765a131b3d503acb0fc53e53fbec)